episodCard.js

```diff
@@ -0,0 +1,57 @@
+// Display all episodCard
+export function episodCard(episodes, search = "") {
+  const rootElem = document.getElementById("root");
+  rootElem.innerHTML = "";
+  for (let i = 0; i < episodes.length; i++) {
+    const dive = document.createElement("div");
+    const pHeader = document.createElement("p");
```

Comment:
if you are using `pHeader` as the variable why not `document.createElement("h1");`